phpBB/assets/javascript/installer.js

```diff
@@ -243,8 +243,21 @@
 	 */
 	function parseMessage(messageJSON) {
 		$('#loading_indicator').css('display', 'none');
+		var responseObject;
 
-		var responseObject = JSON.parse(messageJSON);
+		try {
+			responseObject = JSON.parse(messageJSON);
+		}
+		catch (err) {
+			if (window.console) {
```

Comment:
I'd rather not output the error via console or alert. Just catch it and reset the polling by default.